.gitignore

```diff
@@ -26,6 +26,9 @@ inbox/test/.cache
 /src
 
 # virtualenv
+venv
+.venv
+virtualenv
```

Comment:
FYI I also snuck in a commit ignoring these common virtualenv folder names in git. It seems to me that the convention this repo follows is to have the virtualenv in a `venv` folder, judging from the isort config: https://github.com/closeio/sync-engine/blob/07e939072b53360b9178ca7fec07ccc20103f8d5/pyproject.toml#L1-L2